examples/5_teach_robot_record_dataset.py

```diff
@@ -39,11 +36,7 @@
 @click.option("--task", default="pick up the remote", help="The task to perform/record")
 @click.option("--backend", default="openai", help="The backend to use", type=click.Choice(["anthropic", "openai"]))
 @click.option("--backend_api_key", default=None, help="The API key for the backend, i.e. OpenAI, Anthropic")
-@click.option("--disable_audio", default=False, help="Disable audio input/output")
-def main(task: str, backend: str, backend_api_key: str, disable_audio: bool) -> None:
-    if disable_audio:
-        os.environ["NO_AUDIO"] = "1"
-
+def main(task: str, backend: str, backend_api_key: str) -> None:
     context = [
```

Comment:
agent = LanguageAgent("model_src"="openai", context="Respond in the following json schema {...}")